### PR TITLE
Added VIA keymap, Updated Product ID, Enabled RGB

### PR DESCRIPTION
**Pull request by littleferris**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ergosaurus/keymaps/default/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/ergosaurus/keymaps/default/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ergosaurus/config.h [hunk 2]
```diff
@@ -20,11 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID 0xBEEF
-#define PRODUCT_ID 0xFEED
+#define VENDOR_ID 0x434B
+#define PRODUCT_ID 0xE590
 #define DEVICE_VER 0x0001
 #define MANUFACTURER reggalicious
 #define PRODUCT ergosaurus
+#define DESCRIPTION An ergo Alice clone from Reggalicious
```
Comment (drashna) [inline keyboards/ergosaurus/config.h:28]: This is deprecated and needs to be removed.
```suggestion
```

File: keyboards/ergosaurus/config.h [hunk 3]
```diff
@@ -58,24 +59,24 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-// #ifdef RGBLIGHT_ENABLE
-//   #define RGB_DI_PIN B6
-//   #define RGBLED_NUM 16
-//   #define RGBLIGHT_HUE_STEP 8
-//   #define RGBLIGHT_SAT_STEP 8
-//   #define RGBLIGHT_VAL_STEP 8
-//   #define RGBLIGHT_LIMIT_VAL 255 /* The maximum brightness level */
-//   #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+ #define RGB_DI_PIN B6
+ #ifdef RGB_DI_PIN
```
Comment (drashna) [inline keyboards/ergosaurus/config.h:63]:
```suggestion
#ifdef RGBLIGHT_ENABLE
#    define RGB_DI_PIN B6
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/ergosaurus/config.h [hunk 4]
```diff
@@ -198,6 +198,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  * MIDI options
```
Comment (fauxpark) [inline keyboards/ergosaurus/config.h:198]: The whole MIDI block here might as well be removed.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**